client/allocrunner/taskrunner/task_dir_hook.go

```diff
@@ -33,6 +42,15 @@ func (h *taskDirHook) Name() string {
 }
 
 func (h *taskDirHook) Prestart(ctx context.Context, req *interfaces.TaskPrestartRequest, resp *interfaces.TaskPrestartResponse) error {
+	if v, ok := req.HookData[TaskDirHookIsDoneDataKey]; ok && v == "true" {
+		fsi := h.runner.driverCapabilities.FSIsolation
```

Comment:
We can move this line to the top of the function and not have to repeat it on line 64